src/serializer/brat.py

```diff
@@ -0,0 +1,250 @@
+import json
+import os
+from collections import defaultdict
+from typing import DefaultDict, Dict, List, Optional, Sequence, Tuple, TypeVar
+
+from pie_datasets.core.dataset_dict import METADATA_FILE_NAME
+from pie_modules.annotations import BinaryRelation, LabeledMultiSpan, LabeledSpan
+from pie_modules.documents import TextBasedDocument
+from pytorch_ie import AnnotationLayer
+from pytorch_ie.core import Document
+from pytorch_ie.core.document import Annotation, BaseAnnotationList
+from pytorch_ie.utils.hydra import serialize_document_type
+
+from src.serializer.interface import DocumentSerializer
+from src.utils import get_pylogger
+
+log = get_pylogger(__name__)
+
+D = TypeVar("D", bound=Document)
+
+
+def serialize_labeled_spans(
```

Comment:
Can you outsource some code to a method `serialize_labeled_span(idx: int, labeled_span: Union[LabeledMultiSpan, LabeledSpan], label_prefix: Optional[str] = None) -> str`? I think this would help to simplify the tests.